### PR TITLE
Update Monitoring documentation

### PR DESCRIPTION
**Pull request by davidmirror-ops**
## Why are the changes needed?
Similar to what's requested on https://github.com/flyteorg/flyte/issues/5205 and while this PR doesn't close that Issue, it adds recommendations on how to make use of the Grafana dashboards, connecting a Prometheus instance, the Service Monitors and the necessary Grafana config. 

## Check all the applicable boxes 

- [X] I updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] All commits are signed-off.

### CODE REVIEW

File: docs/deployment/configuration/monitoring.rst [hunk 1]
```diff
@@ -85,15 +85,54 @@ Use Published Dashboards to Monitor Flyte Deployment
...
-        - ControlPlane/Flyteadmin dashboards published @ `13981 <https://grafana.com/grafana/dashboards/13981>`__
+- **System Dashboards**: Dashboards that are useful for the system maintainer to investigate the status and performance of their Flyte deployments. These are further divided into:
+        - `DataPlane/FlytePropeller <https://grafana.com/grafana/dashboards/13979>`__: execution engine status and performance.
+        - `ControlPlane/Flyteadmin<https://grafana.com/grafana/dashboards/13981>`__: API-level monitoring.
+
+The corresponding JSON files for each dashboard are also located at ``deployment/stats/prometheus``.
+
+.. note::
+
+    The above mentioned are basic dashboards and do no include all the metrics exposed by Flyte.
```
Comment (neverett) [inline docs/deployment/configuration/monitoring.rst:101]:
```suggestion
    The dashboards mentioned are basic dashboards and do not include all the metrics exposed by Flyte.
```

File: docs/deployment/configuration/monitoring.rst [hunk 2]
```diff
@@ -85,15 +85,54 @@ Use Published Dashboards to Monitor Flyte Deployment
...
+    Feel free to use the scripts provided `here <https://github.com/flyteorg/flyte/tree/master/stats>`__ to improve and -hopefully- contribute the improved dashboards.
+
+To consume the dashboards, it's recommended to install and configure the Prometheus operator as described in `their docs <https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md>`__.
```
Comment (neverett) [inline docs/deployment/configuration/monitoring.rst:104]:
```suggestion
To consume the dashboards, we recommend installing and configuring the Prometheus operator as described in `their docs <https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md>`__.
```

**Review by neverett — DISMISSED** [review]
Two small suggestions, otherwise LGTM!

**Comment by davidmirror-ops** [comment]
@neverett I fixed some errors and proposed a slightly better format for the instructions. Thank you!

**Review by neverett — APPROVED** [review]

**Pull request merged**